bin/projects/dbatools/dbatools/Parameter/DbaInstanceParameter.cs

```diff
@@ -450,6 +452,26 @@ public DbaInstanceParameter(object Input)
                         throw new PSArgumentException("Failed to interpret input as Instance: " + Input, e);
                     }
                     break;
+                case "microsoft.sqlserver.management.registeredservers.registeredserver":
```

Comment:
Please create a C# unit tests in dbatools.test.csproj that covers at the very least this entire case statement in the `DbaInstanceParameter(object Input)` constructor overload.